api-client/src/main/java/com/desk/java/apiclient/model/Links.java

```diff
@@ -74,4 +74,8 @@ public Link getPrevious() {
     public Link getNext() {
         return next == null ? new Link() : next;
     }
+
+    void setSelf(Link self) {
```

Comment:
I was trying to avoid adding a setter for these base links. If this is only for the purpose of the test below I think we should just mock the `userLinks.getSelfLink()` for the test.